pkg/config/config_template.yaml

```diff
@@ -170,6 +170,13 @@ api_key:
 #
 # forwarder_retry_queue_max_size: 30
 
+## @param forwarder_retry_queue_all_payloads_max_size - integer - optional - default: 0
+## It defines the maximum size in bytes of all the payloads in the forwarder's retry queue.
+## If "forwarder_retry_queue_all_payloads_max_size" is greater than 0, 
+## "forwarder_retry_queue_max_size" is ignored.
+#
+# forwarder_retry_queue_all_payloads_max_size: 0
```

Comment:
I think the `all` is redundant, what about `forwarder_retry_queue_payloads_max_size` ?
Also we should add suggest a sane default for this (60MB ?).

Knowing most client don't set this, I think we should set `forwarder_retry_queue_max_size` to zero as a default and use `forwarder_retry_queue_all_payloads_max_size` by default. This plus a deprecating message to push users to use the new and better setting.

Reply:
`all` is removed (I removed `all` from golang variable names but did not update this part).

Reply:
> Knowing most client don't set this, I think we should set forwarder_retry_queue_max_size to zero as a default and use forwarder_retry_queue_all_payloads_max_size by default

I would like to avoid increasing too much the memory usage in this PR. If I set `forwarder_retry_queue_all_payloads_max_size` to something close to what we have when using the default value of `forwarder_retry_queue_max_size` there is no win.